lib/Common/PlatformAgnostic/Arrays.h

```diff
@@ -0,0 +1,35 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+namespace PlatformAgnostic
+{
+namespace Arrays
+{
+    class Utility
+    {
+#ifndef ENABLE_GLOBALIZATION
+        class ArraysLocale
```

Comment:
nit: I think a name like `LocalizedArray` or `ArrayLocalization` is a bit better?

Reply:
Definitely